### DISCUSSION TITLE
Transfer List Input - Search should be implemented on target list

### DISCUSSION THREAD

**Discussion by madebyjono** [formkit/formkit#1373] [Ideas]
Target list can be just as long as  / contains all the initial values of the source-list. Eg, 200 - 400+ items in the target-list.
Searching for a specific item to remove from the target list will be a lot of work, scrolling and visually checking each one. 
It makes sense to have search for both source and target list - or at least make it an option. 

<3 Formkit!

**Comment by sashamilenkovic** [comment]
@madebyjono This available now at FormKit Pro v0.126.1 via. the `target-searchable` prop
